lib/services/swagger-api-service.js

```diff
@@ -70,22 +70,56 @@ function swaggerFactory(
         }).value();
     }
 
+    function _sortQuery(query){
+        var sortParams = new Object();
+        var tempStr = query.raw;
+        var matchOperation = tempStr.match(/-/g)? true:false;
```

Comment:
I think this will have problems if there happens to be a `-` in the property you're querying for.  I'd suggest using a regex capture to split `sortBy` and `dsc`.
```
var match = query.raw.match(/([-+]{0,1})(.+)/)
sortParams.sortBy = match[1];
sortParams.dsc= match[2];
```